packaging/installer/netdata-updater.sh

```diff
@@ -210,6 +210,13 @@ tmpdir=
 
 trap cleanup EXIT
 
+# Random sleep to aileviate stampede effect of Agents upgrading
+# and disconnecting/reconnecting at the same time (or near to).
+# But only we're not a controlling terminal (tty)
+if [ ! -t 1 ]; then
+  sleep $((RANDOM % (3600 - 1800 + 1) + 1800))s
```

Comment:
Unless I'm mistaken, this results in a range of 30-60 minutes for the delay. Is there some reason not to just make it 0-60 minutes instead if we're putting 60 minutes as the upper bound?

Reply:
Its called "middle ground" I met you halfway. 😀

Reply:
> Unless I'm mistaken, this results in a range of 30-60 minutes for the delay.

this is not clear from the code, i sugges to add it as a code ([min, max])

Reply:
@prologic I think I might have been unclear then in what I was saying. I still think the lower limit should be 0 seconds, I just believe that we should spread across a much longer period of time than the 10s that your original code did.

If we really are just going to do a 30 minute range, then that should be 0-30 minutes, not 30-60.